### ISSUE TITLE
Enhancing our CI

### ISSUE THREAD

**Issue by MartinHinz** [ISAAKiel/ISAAKiel.github.io#5]
Just one idea to enhance our Coporate Identity and presence: Stickers!
Look at the following, and comment, please!

Costs at flyeralarm: removable sticker, 8.5x5.5 cm, 25,44 € for 100 Stickers + Shipping. If we order more, it will be cheaper per item.

[isaak_sticker.pdf](https://github.com/ISAAKiel/ISAAKiel.github.io/files/1526735/isaak_sticker.pdf)

**Comment by nevrome** [comment]
In the developer community hex stickers are very popular. 

http://hexb.in/
https://www.redbubble.com/shop/tidyverse+stickers

It would be great if [somebody](https://www.youtube.com/watch?v=G-BOTFmimoA&feature=youtu.be&t=4) had this already in mind when we discussed the logo.

**Comment by MartinHinz** [comment]
Indeed. And learn from Hadley is is learning to win (free translated GDR-Slogan). But unfortunately at least Flyeralarm does not offer hex-stickers that are removeable (I have to give my laptop back probably somewhen). Also, our slogan has to fit in somewhere!?

**Comment by MoritzMennenga** [comment]
They produce nearly everything, I think with a call we can ask if the can do removeable hex-stickers (I have to call them later anyway).

**Comment by MoritzMennenga** [comment]
[isaak_sticker.pdf](https://github.com/ISAAKiel/ISAAKiel.github.io/files/1527175/isaak_sticker.pdf)
Not well orientated, but something like that?

**Comment by MartinHinz** [comment]
That is definitely something we can work upon! Cool! Worth reactivating Janine in the end, after we decided on a option?

**Comment by MoritzMennenga** [comment]
wir-machen-druck can make hex-stickers 100 = 22,5€, but they don't guarantee for residue-free removal.

**Comment by MartinHinz** [comment]
My poor Laptop. Have to ask Christoph if this is acceptable (sitting currently right next to me, but can not speak freely at the moment :-) )

**Comment by MoritzMennenga** [comment]
I asked them again, if they are made to be removed -  if they won't guarantee it for insurance reasons.
-> we can get a sample peace

**Comment by MoritzMennenga** [comment]
They will send me some samples

**Comment by MoritzMennenga** [comment]
The sampels are on my old notebook. We will see which one is removabale next year.

**Comment by MartinHinz** [comment]
Iwantzomepicturez

**Comment by MoritzMennenga** [comment]
I tried to remove the sample stickers.... Now I have some non-removable and some bad-removable on my old pc... but - there was one very easy to remove.
The prices are (ca. 4,7 x 5,5 cm hex):
33,63 -> 300 pieces
26,74 -> 200
19,86 -> 100
I think we should order them!
I made an example for the layout, maybe we can discuss this on Thursday

**Comment by MartinHinz** [comment]
My version of Moritz suggestion:
[isaak_sticker_mh.pdf](https://github.com/ISAAKiel/ISAAKiel.github.io/files/1796983/isaak_sticker_mh.pdf)
also as svg available, but not currently uploadable here. Have been distributed via email.

**Comment by MoritzMennenga** [comment]
The voting will close today 9 pm utc+1!
I will prepare the data for the printing this evening and will hand it in. I think it will be printed until next week.

**Comment by nevrome** [comment]
My vote: Martins Version (isaak_sticker_mh.pdf) * 100 in 4,7 x 5,5 cm.

**Comment by MartinHinz** [comment]
For the sake of clarity: I vote for my version ;-)

OK with Clemens proposal for size and number.

Am Freitag, den 09.03.2018, 05:32 -0800 schrieb Clemens Schmid:

**Comment by MartinHinz** [comment]
For Dirk: 
===snip===
Hey, super! Ich bin für isaak_sticker_mh

Beste Grüße,

Dirk
===snip===

**Comment by MartinHinz** [comment]
For Carolin (without an e):
===snip===
Dann vote ich mal schnell:

Ich bin für die isaak_sticker_mh Version ohne Serifen. :)

Wäre ja super, wenn das zur CAA noch klappen würde.

Grüße,
Caro
===snip===

**Comment by MoritzMennenga** [comment]
I like Martins sticker, too

**Comment by quassinja** [comment]
I like Martins Sticker

**Comment by MartinHinz** [comment]
Nils MS wrote:
===snip===

Ich ebenso!

Nils

> Am 09.03.2018 um 14:55 schrieb Dirk Seidensticker <dirk.seidensticker@gmail.com>:
> 
> Hey, super! Ich bin für isaak_sticker_mh
> 
> Beste Grüße,
> 
> Dirk

===snip===

**Comment by MartinHinz** [comment]
Here is a background for you laptops in case you might need it in conferences (eg. CAA 2018)...
![isaak_background](https://user-images.githubusercontent.com/3285169/37640741-4cfb8b18-2c17-11e8-95e4-e28b140030df.png)
